src/main/php/scriptlet/HttpScriptletRequest.class.php

```diff
@@ -281,6 +281,14 @@ public function getSessionId() {
    * @return string
    */
   protected function importParam($value) {
+
+    // Handle file uploads - these are represented as arrays containg elements for each
+    // file being uploaded
```

Comment:
This was only a fast solution that we write with @kiesel  to fix the problem with multiple file uploads. Calls with ?order[]=size&order[]=name don't run in the problem. The error eccource only if the values are multiple arrays like in files.